hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/ObservableResultsTest.java

```diff
@@ -63,20 +63,19 @@
     private TestObserver testObserver;
     private Observable<Long> testObservable;
     private UUID registrationId;
-    private Set<Observable<?>> usedObservables;
 
     @Before
     public void before() {
-        usedObservables = new HashSet<>();
         observableName = randomName();
         testObserver = new TestObserver();
-        testObservable = getObservable(observableName);
+        testObservable = jet().getObservable(observableName);
         registrationId = testObservable.addObserver(testObserver);
     }
 
     @After
-    public void after() {
-        usedObservables.forEach(Observable::destroy);
+    public void after() throws Exception {
```

Comment:
method should have `@Override` if it's overriding, or you can just use another method name